platform-sdk/swirlds-common/build.gradle.kts

```diff
@@ -15,37 +15,40 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("org.gradle.java-test-fixtures")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
```

Comment:
intentional?

Reply:
I assume that files were never touched after we introduced the new formatting and spotless.